### DISCUSSION TITLE
Add "Playback Events" On/Off shortcut to main screen

### DISCUSSION THREAD

**Discussion by arnovandyk** [artisan-roaster-scope/artisan#856] [Ideas]
Hi! 
I don't know if this would help anyone else, or if there is a way that I can program an extra button to appear on my main screen while roasting myself, but it would help me immensely if I'm able to switch the "playback events" on or off with one click, instead of needing to go through the steps Roast>Background>Playback tick.
Thank you !

**Comment by MAKOMO** [comment]
In Artisan v2.6.0 PlayBack Events can be toggled using the j key.

See the [v2.6.0 release notes](https://artisan-roasterscope.blogspot.com/2022/03/artisan-v260.html).

You can as well define a custom event button with using the  `Artisan Command` `playbackmode` to control the PlayBack Events mode.

**Reply by arnovandyk** [reply]
How would I go about the defining of a custom button? My setup only has a touch screen (not a very great one) hence wanting a button to press.

**Comment by arnovandyk** [comment]
![image](https://user-images.githubusercontent.com/96994719/160117576-ba734065-1984-409f-92d6-219ca1aff692.png)

Where can I find the Documentation to make the button work? Sorry for asking, not exactly a  programmer!

**Reply by MAKOMO** [reply]
Please upgrade Artisan to the current v2.6.0!

**Comment by MAKOMO** [comment]
There is lots of documentation on the Artisan Blog and in the Events dialogs help page (just press the Help button).

Here is a configuration which toggles PlayBack OFF and ON (replay by bean temperature).

<img width="906" alt="Screenshot 2022-03-25 at 13 23 54" src="https://user-images.githubusercontent.com/13909011/160120431-38beb27c-2f7b-4a3c-90fe-0b34df2db995.png">

**Comment by arnovandyk** [comment]
Thank You!
